### PR TITLE
53 docs wiff mzml

### PR DESCRIPTION
**Pull request by jalew188**

### CODE REVIEW

File: alpharaw/mzml.py [hunk 1]
```diff
@@ -10,10 +10,38 @@
 
 
 class MzMLReader(MSData_Base):
+    """
+    Load mzml file as `:class:`MSData_Base` structure.
+    Register "mzml" in :data:`ms_reader_provider`.
```
Comment (mschwoer) [inline alpharaw/mzml.py:15]: This registering is strictly speaking not a part of this this class, but of the containing module (and should be refactored, as in alphabase)

File: alpharaw/mzml.py [hunk 2]
```diff
@@ -10,10 +10,38 @@
...
+
+    Parameters
+    ----------
+    filename : str
```
Comment (mschwoer) [inline alpharaw/mzml.py:19]: a parameter `filename` containing a file path? should we make `filename` deprecated and add a parameter "file_path"?
also:  absolute? relative? :-)

File: alpharaw/mzml.py [hunk 3]
```diff
@@ -10,10 +10,38 @@
...
+        Spectrum information dict.
+    """
+
     def _import(
         self,
         filename: str,
-    ):
+    ) -> dict:
+        """
+        Re-implementation of :func:`alpharaw.MSData_Base._import`.
```
Comment (mschwoer) [inline alpharaw/mzml.py:33]: why Re-implementation? isn't this a implementation specific to the mzml format? also, maybe add smth like "See documentation of superclass for details."

File: alpharaw/mzml.py [hunk 4]
```diff
@@ -96,6 +124,19 @@ def _import(
 
 
 def parse_mzml_entry(item_dict: dict) -> tuple:
+    """
+    Parse mzml entries from pyteomics extracted items.
```
Comment (mschwoer) [inline alpharaw/mzml.py:128]: would adding an example help here?

File: alpharaw/sciex.py [hunk 5]
```diff
@@ -32,6 +32,19 @@ def __init__(self, centroided: bool = True, save_as_hdf: bool = False, **kwargs)
         self.file_type = "sciex"
 
     def _import(self, _wiff_file_path: str) -> dict:
+        """
+        Re-implementation of :func:`alpharaw.MSData_Base._import`.
```
Comment (mschwoer) [inline alpharaw/sciex.py:36]: (see above)

**Review by mschwoer — APPROVED** [review]

File: alpharaw/mzml.py (hunk 1 above)
Comment (jalew188) [reply alpharaw/mzml.py:15]: Yes, we still need to remind users that they can also access this class by using the provider.

File: alpharaw/mzml.py (hunk 3 above)
Comment (jalew188) [reply alpharaw/mzml.py:33]: _import is a interface. `Implementation` would be better

File: alpharaw/mzml.py (hunk 4 above)
Comment (jalew188) [reply alpharaw/mzml.py:128]: No, not for mzml ...

File: alpharaw/sciex.py (hunk 5 above)
Comment (jalew188) [reply alpharaw/sciex.py:36]: Yep

File: alpharaw/mzml.py (hunk 1 above)
Comment (jalew188) [reply alpharaw/mzml.py:15]: I will use `register_readers` for all readers in the next PR.

**Pull request merged**